A58-client-side-weighted-round-robin-lb-policy.md

```diff
@@ -205,39 +202,31 @@ XdsLbPolicyRegistry::XdsLbPolicyRegistry() {
 }
 ```
 
-In the xDS configuration for the client, `client_side_weighted_round_robin` is wrapped in `wrr_locality` as a child policy. Below is an example xDS configuration with `client_side_weighted_round_robin`. The config includes `round_robin` as a backup policy to fallback in case the client does not support `client_side_weighted_round_robin`. On the client the new xDS LB hierarchy in [gRFC A52] creates the gRPC LB policy `weighted_round_robin` as the endpoint-picking policy.
+In the xDS configuration for the client, `client_side_weighted_round_robin` is wrapped in `wrr_locality` as a child policy. Below is an example xDS configuration with `client_side_weighted_round_robin`. The config includes `round_robin` as a backup policy to fallback in case the client does not support `client_side_weighted_round_robin`. On the client the new xDS LB hierarchy in [gRFC A52][A52] creates the gRPC LB policy `weighted_round_robin` as the endpoint-picking policy.
 
-```json
+```textproto
 { // Cluster
   name: "..."
   type: EDS
-  load_balancing_policy: {
-    policies: [
-      typed_extension_config: {
+  load_balancing_policy {
+    policies {
+      typed_extension_config {
         name: "..."
-        typed_config: {
-          "@type": "type.googleapis.com/envoy.extensions.load_balancing_policies.wrr_locality.v3.WrrLocality"
-          endpoint_picking_policy: {
-            policies: [
-              typed_extension_config: {
-                name: "..."
-                typed_config: {
-                  "@type": "type.googleapis.com/envoy.extensions.load_balancing_policies.client_side_weighted_round_robin.v3.ClientSideWeightedRoundRobin"
-                  type_url: "..."
-                  fields: [
-                    {
-                      key: "enable_oob_load_report"
-                      value: false
-                    },
-                    // …
-                  ]
+        typed_config {
+          [type.googleapis.com/envoy.extensions.load_balancing_policies.wrr_locality.v3.WrrLocality] {
+          endpoint_picking_policy {
```

Comment:
This (and everything inside of it) should be indented one more level.

Reply:
Re-formatted.